packages/onboarding/src/step-container-v2/components/buttons/BackButton/style.scss

```diff
@@ -1,5 +1,6 @@
 .step-container-v2__back-button {
 	--wp-components-color-accent: var(--color-neutral-100);
+	--color-link: var(--color-neutral-100);
```

Comment:
The WP component library says that links are supposed to be blue. So having black links in the onboarding flow really is a special case.